internal/application/documentation.go

```diff
@@ -148,11 +167,13 @@ func (f *DocumentationFilter) annotateApplication(app *yaml.RNode) error {
 	// This is used to ensure even empty (and otherwise omitted) fields can be
 	// included for documentation purposes.
 	required := map[string]string{
-		"resources":  "",
-		"parameters": "resources",
-		"scenarios":  "parameters",
-		"objectives": "scenarios",
-		"":           "objectives",
+		"resources":   "",
+		"parameters":  "resources",
+		"scenarios":   "parameters",
+		"objectives":  "scenarios",
+		"stormForger": "objectives",
+		"ingress":     "stormForger",
```

Comment:
This should stay consistent with the declaration order in the `Application` struct since that is what the formatter will use to order the YAML fields.

Reply:
By way of the completely non-obvious [YAML field ordering initializer](https://github.com/thestormforge/optimize-controller/blob/master/internal/experiment/generation/generation.go#L58).